lib/policy_machine_storage_adapters/in_memory.rb

```diff
@@ -28,6 +28,10 @@ class InMemory
       # The results are paginated via will_paginate using the pagination params in the params hash
       # The find is case insensitive to the conditions
       define_method("find_all_of_type_#{pe_type}") do |options = {}|
+        if options.delete(:where)
```

Comment:
You need this in the neography adapter too. Also, this is going to be awkward for integration testing in in-memory mode. If you'd like, you could make this more limited (an `attribute_containing` key or something like that) and I could add it to the other adapters. Part of the reason we don't expose active_record functionality directly is to maintain compatibility with non-sql graph databases.